backend/websocketConnect.py

```diff
@@ -0,0 +1,39 @@
+from fastapi import FastAPI, WebSocket, WebSocketDisconnect
+import asyncio
+
+biostepFeedback = FastAPI()
+
+@biostepFeedback.websocket("/ws")
+async def websocket_endpoint(websocket: WebSocket):
+    print("Waiting for Connection")
+    
+    try:
+        await websocket.accept()
+        print("Connection accepted")
+    except asyncio.TimeoutError:
+        print("Connection timed out")
+        return
+    except OSError as e:
+        print(f"Network error occurred: {e}")
+        return
+    except Exception as e:
+        print(f"Error Occurred: {e}")
+        return 
+
+    while True:
```

Comment:
This looks good because it keeps the channel open and allows for real-time communication between server and client, however, it might block other operations from running. Consider refactoring this to cater for multiple clients per time